packages/react/src/components/Radio/Radio.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import type { Meta, StoryFn, StoryObj } from '@storybook/react';
+
+import { Radio } from '.';
+
+type Story = StoryObj<typeof Radio>;
+
+export default {
+  title: 'Kjernekomponenter/Radio',
+  component: Radio,
+  parameters: {
+    status: {
+      type: 'beta',
+      url: 'http://www.url.com/status',
+    },
+  },
+} as Meta;
+
+// Simple story
+// First story is the one displayed by <Preview /> and used for <Controls />
+export const Preview: Story = {
+  args: {
+    children: 'You created the Radio component!',
+    description: 'Description',
+  },
+};
+
+// Function story
+// Use this story for listing our different variants, patterns with other components or examples usage with useState
+export const Composed: StoryFn<typeof Radio> = () => (
+  <fieldset>
+    <legend>
+      What killed the radio star?
+      <span
+        role='img'
+        aria-label='guitar emoji'
+      >
+        🎸
+      </span>
+    </legend>
+    <ul style={{ listStyle: 'none' }}>
```

Comment:
In a notable exception, Safari will not recognize an ordered or unordered list as a list in the accessibility tree if it has a `list-style` value of `none`. Since this is a story that might not be an issue _(our examples should be cross-browser accessible)_, but to stay accessible on Safari we need to add `role="list"` explicitly.

Reply:
Good point! I removed usage of `ul` and `li` in this story as it should work without! We should however add this to our `List` component!

Reply:
I've made a separate issue on this #670